libs/util/shared/src/lib/elasticsearch/elasticsearch.service.ts

```diff
@@ -196,37 +190,59 @@ export class ElasticsearchService {
         }
   }
 
-  buildAutocompletePayload(query: string): Observable<EsSearchParams> {
-    return this.uiConf.pipe(
-      map((config) => {
-        const template = config.mods.search.autocompleteConfig
-        return {
-          ...template,
-          _source: [
-            ...template._source.filter((source) => source !== 'uuid'),
-            'uuid',
+  buildAutocompletePayload(query: string): EsSearchParams {
+    const autocompleteConfig = {
+      query: {
+        bool: {
+          must: [
+            {
+              multi_match: {
+                query: '',
+                type: 'bool_prefix',
+                fields: [
+                  'resourceTitleObject.${searchLang}',
+                  'resourceAbstractObject.${searchLang}',
+                  'tag',
+                  'resourceIdentifier',
+                ],
+              },
+            },
           ],
-          query: {
-            ...template.query,
-            bool: {
-              ...template.query.bool,
-              must: [
-                this.addTemplateClause('n'),
-                {
-                  multi_match: {
-                    ...template.query.bool.must[0].multi_match,
-                    query,
-                  },
-                },
-                ...template.query.bool.must.filter(
-                  (clause) => !('multi_match' in clause)
+        },
+      },
+      _source: ['resourceTitleObject'],
+      from: 0,
+      size: 20,
+    }
+    return {
```

Comment:
if there is no more a config, then you can get rid of the cascade spread operator, just build the correct object once.
use the tests to be sure you created exactly the same object as before

Reply:
Good point, completely missed that, should be better now.